ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -52,6 +52,9 @@ export const AnacondaWizard = ({ onAddErrorNotification, toggleContextHelp, hide
     const [showPassphraseScreen, setShowPassphraseScreen] = useState(false);
     const [storageScenarioId, setStorageScenarioId] = useState(window.sessionStorage.getItem("storage-scenario-id") || getDefaultScenario().id);
 
+    // True if using the quit button will reboot the system
+  const isQuitReboot = conf["Installation System"].type === "BOOT_ISO";
```

Comment:
Can you maybe rename the isQuiteReboot to something more self explanatory? isBootIso? or invert the logic and make it isLiveMedia? 

The you can also improve the comment so that people understand what's going on here./
/* On live media rebooting the system will actually shut it off */

Or whatever the case is. And if you can support it with some link so a documentation that would be idea.